src/node-red/index.njk

```diff
@@ -135,6 +135,60 @@ sitemapPriority: 0.8
     See more customer stories {% include "components/icons/chevron-right-sm.svg" %}
 </a>
 
+<!--Flexible Deployment with FlowFuse-->
+<div class="w-full ff-bg-light">
+    <div class="bg-gray-100 text-center text-teal-50 px-6 pt-3 md:pt-8 pb-16 md:pb-0 text-lg flex-col flex-row items-center justify-center gap-1 hover:no-underline flex-wrap">
+        <div class="text-center mt-6 md:mt-4 mb-14">
+            <h2>Flexible Deployment Options with <span class="text-red-600">FlowFuse</span></h2>
+            <div class="w-full text-center text-gray-800 font-light">Run and manage <span class="text-red-600 font-medium">Node-RED</span> where it's most helpful for your business</div>
+        </div>
+        <div class="container m-auto max-w-screen-lg">
+            <div class="grid md:px-6 md:px-0 grid-cols-1 md:grid-cols-3 gap-12 md:gap-5 max-w-md md:max-w-none mx-auto">
+                <div class="deployment-card">
+                    <div>
+                        <div class="title">
+                            <h4>
+                                <span class="text-gray-600">FlowFuse Cloud</span></br>
+                            </h4>
+                        </div>
+                        <p class="mb-0 text-left">
+                            The easiest way to start building Node-RED applications—no installation required. Simply sign up and gain immediate access to the editor.
+                        </p>
+                    </div>
+                    <a class="md:self-end ff-btn ff-btn--primary-outlined uppercase align-baseline w-full mt-3" href="/pricing/">MORE INFO</a>
```

Comment:
Is this the correct CTA?

Reply:
It's linking to the pricing page, not the sign-up page. That's why I thought of "More info" instead of sign-up.

My thinking here is that we already have the sign up button in the hero section and the main nav, and this section includes the word "business", maybe someone could be interested in a different plan? Team or enterprise...

Do you think it should link to the sign-up page too?